usecases/admin/generate-invite.md

```diff
@@ -4,75 +4,77 @@
             <strong>Name</strong>
         </td>
         <td>
-            lorem pisum
+            Generate Invite
         </td>
     </tr>
     <tr>
         <td>
             <strong>Actor</strong>
         </td>
         <td>
-            lorem pisum
+            Admin
         </td>
     </tr>
     <tr>
         <td>
-            <strong>Description</strong>            
+            <strong>Description</strong>
         </td>
         <td>
-            lorem pisum
+            The actor may generate invitation codes for new users to register with.
         </td>
     </tr>
     <tr>
         <td>
             <strong>Pre-condition</strong>
         </td>
         <td>
-            lorem pisum
+            <ul>
+                <li>Actor is loggedin.</li>
+                <li>System requires invitation codes.</li>
```

Comment:
Optional Server Setting: System requires invitation codes for new users to register